parsl/tests/test_data/test_file_apps.py

```diff
@@ -1,74 +1,60 @@
-import os
-
 import pytest
 
-import parsl
 from parsl.app.app import bash_app
 from parsl.data_provider.files import File
-from parsl.tests.configs.local_threads import config
 
 
 @bash_app
-def cat(inputs=[], outputs=[], stdout=None, stderr=None):
-    infiles = ' '.join([i.filepath for i in inputs])
-    return """echo {i}
-    cat {i} &> {o}
-    """.format(i=infiles, o=outputs[0])
+def cat(inputs=(), outputs=(), stdout=None, stderr=None):
+    infiles = " ".join(i.filepath for i in inputs)
+    return f"cat {infiles} &> {outputs[0]}"
 
 
-@pytest.mark.usefixtures('setup_data')
 @pytest.mark.staging_required
-def test_files():
-
-    if os.path.exists('cat_out.txt'):
-        os.remove('cat_out.txt')
-
-    fs = [File('data/' + f) for f in os.listdir('data')]
-    x = cat(inputs=fs, outputs=[File('cat_out.txt')],
-            stdout='f_app.out', stderr='f_app.err')
+def test_files(setup_data):
+    fs = sorted(str(setup_data / f) for f in setup_data.iterdir())
+    fs = list(map(File, fs))
+    x = cat(
+        inputs=fs,
+        outputs=[File(str(setup_data / "cat_out.txt"))],
+        stdout=str(setup_data / "f_app.out"),
+        stderr=str(setup_data / "f_app.err"),
+    )
+    x.result()
     d_x = x.outputs[0]
-    print(x.result())
-    print(d_x, type(d_x))
+    data = open(d_x.filepath).read().strip()
+    assert "1\n2" == data, "Per setup_data fixture"
 
 
 @bash_app
-def increment(inputs=[], outputs=[], stdout=None, stderr=None):
+def increment(inputs=(), outputs=(), stdout=None, stderr=None):
     # Place double braces to avoid python complaining about missing keys for {item = $1}
```

Comment:
I think this comment can go away too? I don't see any double braces

Reply:
Removed in a follow-up PR.